docker-cli.yaml

```diff
@@ -0,0 +1,69 @@
+package:
+  name: docker-cli
+  version: 24.0.6
+  epoch: 0
+  description: The Docker CLI
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - bash
+      - btrfs-progs-dev
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - containerd
+      - coreutils
+      - go
+      - libseccomp-dev
+      - libtool
+      - linux-headers
+      - lvm2-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/docker/cli
+      tag: v${{package.version}}
+      expected-commit: ed223bc820ee9bb7005a333013b86203a9e1bc23
+
+  - runs: |
+      export DISABLE_WARN_OUTSIDE_CONTAINER=1
+
+      mkdir -p "$GOPATH/src/github.com/docker/"
+      ln -sf /home/build "$GOPATH"/src/github.com/docker/cli
+      LDFLAGS="" make VERSION=${{package.version}} dynbinary
+
+      install -Dm755 build/docker-linux-* ${{targets.destdir}}/usr/bin/docker
+
+  - runs: |
+      make manpages
+      install -Dm644 man/man1/* -t ${{targets.destdir}}/usr/share/man/man1/
+
+  # Docker is vehemenetly against stripping the resulting binary
+  # Ref: https://github.com/moby/moby/blob/d8a51d2887cbc465ab8d76ed98f7a86996ab3c22/project/PACKAGERS.md#stripping-binaries
+  # - uses: strip
+  - runs: |
```

Comment:
why does yam want this?

Reply:
yam doesn't seem to understand empty comment blocks, and squishes this comment block into the previous node resulting in bad yaml.

if there's a better way to warn us from `strip` I'm all 👂!

Reply:
ref: https://github.com/wolfi-dev/os/actions/runs/6253508252/job/16979074829?pr=5769